backend_model/Comment_post.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose")
+
+// schema for the comment
+const commentSchema = mongoose.Schema({
+        title: {
+            type: String,
+            required: false
+            
+        },
+        content: {
+            type: String,
+            required: true
+        },
+        author: {
```

Comment:
Is this supposed to be `author` or `post`?

Reply:
it is author

Reply:
And the `author` is pointing to the `post` schema?

Reply:
I adjusted it, now there is comment and content.